neurots/extract_input/from_neurom.py

```diff
@@ -14,92 +14,184 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import warnings
 
 import neurom as nm
 import numpy as np
 from neurom import stats
+from neurom.check.morphology_checks import has_apical_dendrite
+from neurom.core.types import tree_type_checker as is_type
+from scipy.integrate import quad
+from scipy.optimize import curve_fit
 
+from neurots.generate.orientations import prob_function
```

Comment:
Needs to be added to this PR, it's now failing to extract.

Reply:
indeed I'll add now